packages/@react-spectrum/s2/src/Disclosure.tsx

```diff
@@ -195,7 +200,31 @@ const chevronStyles = style({
   flexShrink: 0
 });
 
+
 function DisclosureHeader(props: DisclosureHeaderProps, ref: DOMRef<HTMLDivElement>) {
+  let {
+    UNSAFE_className,
+    UNSAFE_style,
+    children
+  } = props;
+  let domRef = useDOMRef(ref);
+  return (
+    <div
+      style={UNSAFE_style}
+      className={(UNSAFE_className ?? '') + style({display: 'flex', alignItems: 'center', gap: 4})}
+      ref={domRef}>
+      {children}
+    </div>
+  );
+}
+
+/**
+ * A header for a disclosure. Contains a heading and a trigger button to expand/collapse the panel.
```

Comment:
Concerned that the header and trigger have the same description. I'm not sure we want to call this a wrapper for the trigger and an interactive element.

Reply:
that's probably leftover from copy pasting. thanks for pointing it out! will update it